### ISSUE TITLE
Comments: Bulk Action

### ISSUE THREAD

**Issue by gwwar** [Automattic/wp-calypso#13896]
We should be able to modify multiple comments at once. This is only available in list view.

![screen_shot_2017-05-16_at_3 54 10_pm_720](https://cloud.githubusercontent.com/assets/1270189/26128999/5d65789a-3a43-11e7-8b26-5b3895f75cca.png)

- Normally we see the action filters at top. When a user clicks a checkbox this is replaced by the action bar.
- Action bar actions change contextually with the active filter
- The toggle all checkbox is available in top left. If we click it all comments will be collapsed and selected.
- When a card is expanded, a checkbox should not be rendered.
- A card cannot be expanded if it's already selected.

**Comment by Copons** [comment]
Closed by https://github.com/Automattic/wp-calypso/pull/14816

**Comment by Copons** [comment]
Reopen to track the non-`CommentFaker` bulk actions.

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on issue count.

I was able to perform bulk action on comments:
![Screen Shot 2021-01-29 at 15 24 50](https://user-images.githubusercontent.com/6549265/106337558-4a607400-6246-11eb-9cdb-8003f751f630.png)

I am proposing to close this issue with the label `Closed - Not Reproducible` as the action seems to have been implemented already.
